spec/integration/unhandled_errors_in_receivers_spec.rb

```diff
@@ -34,16 +43,21 @@ def to_json
     end
 
     it "an error in one receiver doesn't prevent others from getting messages" do
-      Pwwka::Transmitter.send_message!({ sample: "payload", has: { deeply: true, nested: 4 }},
-                                       "pwwka.testing.foo")
+      Pwwka::Transmitter.send_message!(payload, "pwwka.testing.foo")
       allow_receivers_to_process_queues
 
       expect(WellBehavedReceiver.messages_received.size).to eq(1)
       expect(ExceptionThrowingReceiver.messages_received.size).to eq(1)
     end
+
+    it "logs the payload in the error in the log" do
+      Pwwka::Transmitter.send_message!(payload, "pwwka.testing.foo")
+      allow_receivers_to_process_queues
+      expect(log_data.string).to match(/ERROR.*error processing message.*#{Regexp.escape(payload.inspect)}/i)
```

Comment:
This failed, before fixing the main production code, thus validating the error this is fixing.